### PR TITLE
Have the sample queue promote best practices w/ good questions

### PR DESCRIPTION
**Pull request by wadefagen**
Tweaks to the sample queue to promote best practices,   Also added more people to the random names list, anyone who has any remove involvement with the queue.  The list is incomplete but was a quick first pass.

### CODE REVIEW

File: components/RandomQueue.js [hunk 1]
```diff
@@ -144,7 +157,7 @@ class RandomQueue extends React.Component {
     // the user is on another tab - otherwise N changes will be flushed to
     // the display at the same time when the user comes back to this tab,
     // which temporarily kills perf
-    this.interval = setInterval(() => this.advanceState(), 5000)
+    this.interval = setInterval(() => this.advanceState(), 15000)
```
Comment (nwalters512) [inline components/RandomQueue.js]: 10s maybe?

File: components/RandomQueue.js [hunk 2]
```diff
@@ -5,10 +5,18 @@ import { ListGroup } from 'reactstrap'
 import Question from './Question'
 
 const STAFF = [
-  'Nathan Walters',
-  'Genna Helsel',
-  'Jordi Paris Ferrer',
-  'Wade Fagen-Ulmschneider',
+  'TA - Nathan Walters',
```
Comment (nwalters512) [inline components/RandomQueue.js]: Not a huge fan of TA/CA prefix - looks a bit cluttered and isn't really reflective of how the Queue actually looks.

File: components/RandomQueue.js [hunk 3]
```diff
@@ -19,22 +27,27 @@ const STUDENTS = [
   'Karlie Kloss',
   'Elon Musk',
   'Grace Hopper',
-  'Steve Jobs',
```
Comment (nwalters512) [inline components/RandomQueue.js:22]: rip 😢

**Comment by nwalters512** [comment]
One thing I noticed in general is that the slightly longer content can wrap to a new line on narrower devices. If we transition from an arrangement of 3 questions with wrapped text to 2 questions with wrapped text and one question with only one line of text, the content below the header jumps around.

File: components/RandomQueue.js (hunk 3 above)
Comment (wadefagen) [reply components/RandomQueue.js:22]: Yeah, one comment was that was a little freaky, sort of agree. :cry:

File: components/RandomQueue.js (hunk 2 above)
Comment (wadefagen) [reply components/RandomQueue.js]: :+1:

**Pull request merged**